src/pa01/snakes_and_ladders.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Jon-Mikkel Korsvik", "Petter Bøe Hørtvedt"
+__email__ = "jonkors@nmbu.no", "petterho@nmbu.no"
+__version__ = "0.0.1"
+
+
+from random import randint
+from random import seed as random_seed
+import numpy as np
+
+
+def check_if_snake_or_ladder(position):
+    """
+    Tuple
+    First argument: position
+    Second argument: the position it leads to.
+    :param position:
+    :return: new position if the position is in the list of snakes and ladders:
+    :if not: original position is returned
+    """
+    list_of_snakes_and_ladders = [
+        (1, 40), (8, 10), (36, 52), (43, 62), (49, 79), (65, 82), (68, 85),
+        (24, 5), (33, 3), (42, 30), (56, 37), (64, 27), (74, 12), (87, 70)]
+    for start_position, end_position in list_of_snakes_and_ladders:
+        if position == start_position:
+            return end_position
+
+    return position
+
+
+def single_game(num_players):
+    """
+    Returns duration of single game.
+
+    Arguments
+    ---------
+    num_players : int
+        Number of players in the game
+
+    Returns
+    -------
+    num_moves : int
+        Number of moves the winning player needed to reach the goal
+    """
+    num_moves = 0
+    list_of_positions = [0]*num_players  # Creates a list with appropriate 0's
+
+    while all([element < 90 for element in list_of_positions]):
```

Comment:
could be relegated to a separate function called `has_won()` or something like that..